api/functions/car-backup-and-index.js

```diff
@@ -0,0 +1,75 @@
+import {
+  S3Client
+} from '@aws-sdk/client-s3'
+
+import { carBackupAndIndex } from '../carpark/backup-and-index.js'
+import parseSqsEvent from '../utils/parse-sqs-event.js'
+
+/**
+ * Get EventRecord from the SQS Event triggering the handler
+ *
+ * @param {import('aws-lambda').SQSEvent} event
+ */
+export function handler (event) {
+  const {
+    BACKUP_ACCOUNT_ID,
+    BACKUP_ACCESS_KEY_ID,
+    BACKUP_SECRET_ACCESS_KEY,
+    BACKUP_CAR_BUCKET_NAME,
+    BACKUP_INDEX_BUCKET_NAME,
+  } = getEnv()
+
+  const record = parseSqsEvent(event)
+  if (!record) {
+    throw new Error('Invalid CAR file format')
+  }
+
+  const destinationBucket = new S3Client({
+    region: 'auto',
+    endpoint: `https://${BACKUP_ACCOUNT_ID}.r2.cloudflarestorage.com`,
```

Comment:
Would be nice if this wasn't specific to cloudflare - can the domain or account+domain be an env var?

Reply:
endpoint is an ENV var instead :)